pages/audit/rules.mdx

```diff
@@ -0,0 +1,119 @@
+---
+layout: 'guide'
+title: 'Avo tracking plan audit rules'
+abstract: 'Learn how to read and fix your tracking plan audit'
+---
+
+import Link from '../../components/Link';
+
+# Audit rules
+
+Following are the static rules Avo tests for in the tracking plan audit. To learn more about how to get started with the tracking plan audit, see <Link passHref href="/audit/quickstart"><a>Quickstart: Audit in 5 minutes</a></Link>.
+
+You will recognize the Tracking Plan Audit by the yellow exclamation point next to the Tracking Plan item in the sidebar. It’s meant to help teams follow their naming convention and surface issues like when a type is missing from a property.
+
+Below is an overview of the reported issues and how you can resolve them.
+
+<center>
+  <img width="400" src={require('../../images/audit/audit.png')} />
+</center>
+
+## Event Rules
+
+### All events have unique names*
+
+Checks if any two event names in the tracking plan conflict with each other. For event name to be unique, no other event can have the same name, independent from their case. Example of conflicting event names:
+
+- "App Opened", "app opened", "app_opened", "app-opened"
+- "Add to Cart", "Add to cart", "add_to_cart", "add-to-cart"
+- "Checkout Started", "checkout started", "checkout-started"
+- "Checkout Completed", "checkout completed", "checkout-completed"
+
+##### How to resolve
+
+Pick the casing that matches the one that is dominant one in your workspace and fix the implementation of those that don't fit your standards. Note that changing your existing tracking might cause issues in downstream tools and metrics. Make sure to discuss with the consumers of the data before making such change.
```

Comment:
Make `Make sure to discuss with the consumers of the data before making such change.` bold? 😀

Reply:
Done in 9f31e1575cd0a6d672211a1dbb9ee81c71692de7